src/client.rs

```diff
@@ -523,6 +523,29 @@ impl Client {
         RequestBuilder::new(Method::Patch, self.url(uri)).with_client(self.clone())
     }
 
+    /// Perform a HTTP request with the given verb using the `Client` connection.
+    ///
+    /// # Panics
+    ///
+    /// This will panic if a malformed URL is passed.
+    ///
+    /// # Errors
+    ///
+    /// Returns errors from the middleware, http backend, and network sockets.
+    ///
+    /// # Examples
+    /// ```no_run
+    /// # #[async_std::main]
+    /// # async fn main() -> surf::Result<()> {
+    /// let client = surf::client();
```

Comment:
```suggestion
    
    /// use surf::Method;
    
```

Reply:
`cargo test` now passes locally after this change. I forgot to run it before making the PR, sorry about that!